README.md

```diff
@@ -9,3 +10,18 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Proof by contradiction:
+
+Assume that two completely connected grpahs A and B, with the same number of nodes, 
+are not isomorphic. This means that there exists an edge from some nodes n_0 to n_1
```

Comment:
Not following you here. The definition doesn't say anything about particular edges.